docs/1.getting-started/2.installation.md

```diff
@@ -22,17 +22,17 @@ Start with one of our starters and themes directly by opening [nuxt.new](https:/
 #### Prerequisites
 
 - **Node.js** - [`v18.0.0`](https://nodejs.org/en) or newer
-- **Text editor** - We recommend [Visual Studio Code](https://code.visualstudio.com/) with the [Volar Extension](https://marketplace.visualstudio.com/items?itemName=Vue.volar)
+- **Text editor** - We recommend [Visual Studio Code](https://code.visualstudio.com/) with the [Volar extension](https://marketplace.visualstudio.com/items?itemName=Vue.volar)
```

Comment:
I think the official name changed here too

Reply:
Yeah, as mentioned in the v2.0.0 release notes, the extension has been renamed from "Volar Language Features (Volar)" to "Vue - Official". So, should we align and replace "Volar" with "Vue - Official" or just mention "official Vue extension"? Or should we keep "Volar" since it's mainly known by this name?